LLMonFHIR/FHIR Display/FHIRResourcesView.swift

```diff
@@ -7,31 +7,39 @@
 //
 
 import ModelsR4
+import SpeziLocalStorage
+import SpeziOnboarding
 import SpeziOpenAI
 import SwiftUI
 
+private enum FHIRMultipleResourceInterpreterConstants {
+        static let storageKey = "FHIRMultipleResourceInterpreter.Cache"
+    }
+
 
 struct FHIRResourcesView: View {
-    @EnvironmentObject var fhirStandard: FHIR
+    typealias MultipleResourceInterpretation = String
+    
+    @AppStorage(StorageKeys.onboardingInstructions) var onboardingInstructions = true
+    
     @State var resources: [String: [FHIRResource]] = [:]
+    @State var allResourcesArray: [FHIRResource] = []
     @State var showSettings = false
+    @State var showMultipleResourcesChat = false
+    @State var interpretingMultipleResources = false
+    @State var error: String?
```

Comment:
The error is currently not used anywhere, we could also just print the error when it is emitted.